src/metabase/cmd.clj

```diff
@@ -24,6 +24,9 @@
             [metabase.util :as u]
             [metabase.util.i18n :refer [trs]]))
 
+(defn- system-exit! [return-code]
+  (System/exit return-code))
```

Comment:
`System/exit` can't be `with-redef`ed.

Reply:
if you still have that branch open that would make a great docstring for why this function exists. that's good context and a good reason for this function to exist

Reply:
@dpsutton, I'll add a docstring there in an immediate PR if this gets green so we merge it as it is.